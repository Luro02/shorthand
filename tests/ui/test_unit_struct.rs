use shorthand::ShortHand;

#[derive(ShortHand)]
struct UnitStruct;

fn main() {}
