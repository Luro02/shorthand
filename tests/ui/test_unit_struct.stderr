error: Unit structs are not supported.
 --> $DIR/test_unit_struct.rs:4:1
  |
4 | struct UnitStruct;
  | ^^^^^^^^^^^^^^^^^^
